src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -106,41 +115,51 @@ class Hovercard extends React.Component {
       },
     };
 
+    let visible;
+    const containerProps = {
+      'aria-describedby': this.tooltipId,
+      className: containerClassName,
+    };
+    const hovercardProps = {
+      withHeader: !!header,
+      className: cx,
+    };
+
+    if (show === true || show === false) {
+      visible = show;
+    } else {
+      visible = this.state.visible;
+      containerProps.onMouseEnter = hovercardProps.onMouseEnter = this.handleToggleOn;
+      containerProps.onMouseLeave = hovercardProps.onMouseLeave = this.handleToggleOff;
+    }
```

Comment:
I think I prefer to see this written as:

```jsx
const visible = show !== undefined ? show : this.state.visible;

if (show === undefined) {
  containerProps.onMouseEnter = hovercardProps.onMouseEnter = this.handleToggleOn;
  containerProps.onMouseLeave = hovercardProps.onMouseLeave = this.handleToggleOff;
}
```

Also maybe we can split these props into their own object like `hoverProps`

```jsx
const visible = show !== undefined ? show : this.state.visible;

const hoverProps = show !== undefined
  ? {}
  : {onMouseEnter: this.handleToggleOn, onMouseLeave: this.handleToggleOff}
```

Probably a bit of a 🚲 🏠 , but it makes it a bit easier for me personally to parse line by line each expression, without having to keep the if else blocks in my mind.